src/v/storage/segment.h

```diff
@@ -166,7 +166,7 @@ class segment {
      * removed already (i.e. marked as a tombstone and closed) then the total
```

Comment:
It looks like this bit about behavior after segment removal may no longer be true?

Reply:
fixed